icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -1209,6 +1209,7 @@ void NumberFormatterApiTest::unitArbitraryMeasureUnits() {
               .unit(MeasureUnit::forIdentifier("pow4-mile", status))
               .unitWidth(UNUM_UNIT_WIDTH_FULL_NAME)
               .locale("en-ZA");
+    lnf.operator=(lnf);  // self-assignment should be a no-op
```

Comment:
(on phone)

Seems like an odd place for this test. A better place would be in the move operator test further down.

Reply:
I just looked for an object with interesting settings that look like they could cause problems in the lnfCopyHelper() and added a copy assignment. If you have a better place, or a more specific test case, please provide a specific location or sample code.